cfgov/v1/preprocessed/css/molecules/text-introduction.less

```diff
@@ -0,0 +1,60 @@
+/* topdoc
+  name: Theme variables
+  family: cfgov-molecules
+  notes:
+    - "The following color and sizing variables are exposed,
+       allowing you to easily override them before compiling."
+  patterns:
+  - name: Colors
+    codenotes:
+      - |
+        @m-text-introduction_rule
+  tags:
+  - less
+*/
+@m-text-introduction_rule: @gray-50;
+
+
+/* topdoc
+  name: Text Introduction
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-text-introduction">
+            <h1 class="m-text-introduction_heading">Heading</h1>
+            <h2 class="m-text-introduction_subheading">Subheading</h2>
+            <p class="m-text-introduction_body">Body content</p>
+            <a class="m-text-introduction_link">Link</a>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-text-introduction
+            h1.m-text-introduction_heading
+            h2.m-text-introduction_subheading
+            p.m-text-introduction_body
+            a.m-text-introduction_link
+  tags:
+    - cfgov-molecules
+*/
+
+.m-text-introduction {
+    &_heading,
+    &_subheading,
+    &_body,
+    &_link {
+        margin-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
+    }
+
+    &__bottom-rule {
+        border-bottom: 1px solid @o-text-introduction_rule;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/bb75a1a4c6f5017433c3de2e12b77c9ee0d04d44